### PR TITLE
Added comments in functions in cgl.h file in my own terms

### PR DESCRIPTION
**Pull request by Bharadwajshivam28**
@Jaysmito101 I have added comments sir in my own words till some lines

### CODE REVIEW

**Comment by Bharadwajshivam28** [comment]
@Jaysmito101 I have added comments sir

**Comment by Bharadwajshivam28** [comment]
@Jaysmito101 Please review it and merge it sir so i will move forward to next lines

**Comment by Jaysmito101** [comment]
As I already said comments should be in the format for doxygen check https://www.cs.cmu.edu/~410/doc/doxygen.html

**Comment by Bharadwajshivam28** [comment]
ok sir

**Comment by Bharadwajshivam28** [comment]
@Jaysmito101 sir please elaborate this line

**Comment by Bharadwajshivam28** [comment]
> As I already said comments should be in the format for doxygen check https://www.cs.cmu.edu/~410/doc/doxygen.html

or just drop a sample sir it will help me alot  @Jaysmito101

**Comment by Jaysmito101** [comment]
@Bharadwajshivam28 check out the link i provided

**Comment by Bharadwajshivam28** [comment]
i checked sir but i am little bit confused

**Comment by Bharadwajshivam28** [comment]
so please drop sample or elborate sir it will be great for me @ja'

**Comment by Bharadwajshivam28** [comment]
getting guidance under you will be great sir @Jaysmito101

**Comment by Jaysmito101** [comment]
@Bharadwajshivam28 A simple example would be:

```
/** @brief Query system for addrinfo
 * 
 * This function queries the system for addrinfo
 * for the given name and port. The addrinfo is
 * stored in the CGL_net_addrinfo struct.
 * 
 * @param name the name of the host
 * @param port the port of the host
 * @param count the number of addrinfo returned (output)
 * @return the addrinfo of the host 
*/
CGL_net_addrinfo* CGL_net_addrinfo_query(const char* name, const char* port, size_t* count);
```

Remember you would have to elaborate the functions purpose properly. Also, I would **strongly** advice going through the entire source of the function, understand it, only then try to describe it. Otherwise it will have mistakes and your effort will be wasted.

**Comment by Bharadwajshivam28** [comment]
ok sir @Jaysmito101

**Comment by Bharadwajshivam28** [comment]
@Jaysmito101 sir please review

**Comment by Bharadwajshivam28** [comment]
its in my own terms and languages and also in correct format

**Comment by Bharadwajshivam28** [comment]
If you agree then will move forward

File: cgl.h [hunk 1]
```diff
@@ -144,34 +144,59 @@ CGL_net_addrinfo* CGL_net_addrinfo_query(const char* name, const char* port, siz
...
+/**
+ * It Connects a socket to a target address.
+ *
+ * @param socket  The socket which we want to connect
+ * @param target  The target address on which we want to connect
+ *
+ * @return IT will return true if connection was successful or else it will give out false
+ */
+bool CGL_net_socket_connect(CGL_net_socket* socket, CGL_net_addrinfo* target)
+{
```
Comment (Jaysmito101) [inline cgl.h:158]: The comment format is more or less correct.
Problems are no brief for the function is given.
Why is a function body added here? its wrong
Check this part of code has already been documented ( https://github.com/Jaysmito101/cgl/commit/00b84d613f0cea61181526f97f7af2cffbc134bf)

**Comment by Bharadwajshivam28** [comment]
ok

**Pull request closed without merging**